src/zone/callback-task.cpp

```diff
@@ -0,0 +1,14 @@
+#include "callback-task.h"
+
+using napa::zone::CallbackTask;
+
+CallbackTask::CallbackTask(Callback callback) 
+    : _callback(std::move(callback)) 
+{ 
+}
+
+void CallbackTask::Execute() 
+{
+    _callback(); 
+}
```

Comment:
One suggestion is to rename `CallTask` to `AsyncCallTask` and `CallbackTask` to `SyncCallTask`

Reply:
Callback in general are called some where else.  Adding this may cause confusion more.